addons/website_crm/controllers/main.py

```diff
@@ -18,3 +18,10 @@ def website_form(self, model_name, **kwargs):
                 request.params['state_id'] = State.search([('code', '=', geoip_state_code), ('country_id.code', '=', geoip_country_code)]).id
 
         return super(WebsiteForm, self).website_form(model_name, **kwargs)
+
+    def insert_record(self, request, model, values, custom, meta=None):
+        # Force a company_id for some models
+        if model.model in ['crm.lead']:
```

Comment:
Could be `if model.model == 'crm.lead':`

Reply:
+1